From 311f91741d0b1f8e58561e61035e8e432074403c Mon Sep 17 00:00:00 2001
From: Andreas Dangel <andreas.dangel@pmd-code.org>
Date: Thu, 5 Sep 2024 14:56:46 +0200
Subject: [PATCH] test case 3 - change in pmd-core

This changes the class PMDVersion which will trigger a
complete reevaluation of all rules by the regression tester.
---
 pmd-core/src/main/java/net/sourceforge/pmd/PMDVersion.java | 1 +
 1 file changed, 1 insertion(+)

diff --git a/pmd-core/src/main/java/net/sourceforge/pmd/PMDVersion.java b/pmd-core/src/main/java/net/sourceforge/pmd/PMDVersion.java
index 19532bd99e..78f61a81cf 100644
--- a/pmd-core/src/main/java/net/sourceforge/pmd/PMDVersion.java
+++ b/pmd-core/src/main/java/net/sourceforge/pmd/PMDVersion.java
@@ -2,6 +2,7 @@
  * BSD-style license; for more info see http://pmd.sourceforge.net/license.html
  */
 
+
 package net.sourceforge.pmd;
 
 import java.io.IOException;
-- 
2.45.2

